### DISCUSSION TITLE
notifications in top menu

### DISCUSSION THREAD

**Discussion by juangdnar** [filamentphp/filament#5123] [Help] [answered]
How can I display the notifications as shown in the top menu of the demo next to the user logo?

![Sin título](https://user-images.githubusercontent.com/115037170/205063168-179a2804-7022-417a-958e-d202b85af99a.png)

**Comment by danharrin** [comment, marked as answer]
https://filamentphp.com/docs/2.x/admin/notifications#database-notifications

**Reply by juangdnar** [reply]
Thanks!!! how should be the structure of the notifications table? is there a migration for it?

**Reply by danharrin** [reply]
yes, its the default Laravel migration table. read the Laravel notifications docs

**Reply by juangdnar** [reply]
Done! Thanks a lot. :)

php artisan notifications:table

https://filamentphp.com/docs/2.x/notifications/database-notifications
